Trust and Compliance: Fiscal provisions (#105)
pretix/pretix-docs: docs/trust/fiscal/austria.md

# Austria

{% include "warning-tax-law.md" %}

> ## Value Added Tax (VAT)

Reviewer: ```suggestion
## Value-added tax (VAT)
```

When selling goods and services in Austria, VAT ("Umsatzsteuer") may apply.
All Austrian VAT rates can be configured in pretix using [tax rules](../../guides/taxes.md), including more complex tax cases (e.g. intra-EU reverse charge transactions).

## Invoicing

### Invoice information

> Austrian requirements on information contained on invoices are defined in [§ 11 UStG 1994](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10004873&FassungVom=2024-08-29&Artikel=&Paragraf=11&Anlage=&Uebergangsrecht=).

Reviewer: ```suggestion
Austrian requirements on information included in invoices are defined in [§ 11 UStG 1994](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10004873&FassungVom=2024-08-29&Artikel=&Paragraf=11&Anlage=&Uebergangsrecht=).
```

Some information can be omitted for invoices of less than €400.

> Invoices created by pretix, if properly configured, can include all necessary information.

Reviewer: ```suggestion
You can configure pretix to include all necessary information in the invoices it generates. 
```

### Invoice numbering

> pretix can create invoice numbers using a user-defined numbering scheme e.g. using a number range per organizer, event, or even order.

Reviewer: ```suggestion
pretix can generate invoice numbers with a user-defined numbering scheme, for example using a number range per organizer, event, or even order.
```

We recommend using consecutive invoice numbers in Austria.

### E-invoicing

> Austria currently does not require sending electronic invoices in a structured format for transactions between businesses (B2B) or businesses and consumers (B2C).

Reviewer: Wissen wir von konkreten Plänen in Österreich, das demnächst zu ändern? Wenn nicht, kann "currently" weg

Reviewer: Okay, dann passt das so

Austria only mandates e-invoices for transactions between businesses and the government (B2G).

> This is currently not supported by pretix.

Reviewer: Haben wir konkrete Pläne, das demnächst zu ändern? Wenn nicht, kann "currently" weg

## pretixPOS

> pretixPOS can be used in compliance with Austrian laws on cash registers ([§ 131b BAO](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10003940&FassungVom=2025-01-27&Artikel=&Paragraf=131b&Anlage=&Uebergangsrecht=), [RKSV](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=20009390)).

Reviewer: ```suggestion
You can use pretixPOS in compliance with Austrian laws on cash registers ([§ 131b BAO](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10003940&FassungVom=2025-01-27&Artikel=&Paragraf=131b&Anlage=&Uebergangsrecht=), [RKSV](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=20009390)).
```

> Use of a cash register is mandatory in Austria for every company with a total yearly turnover of at least € 15,000 and a yearly cash turnover of at least € 7,500.

Reviewer: ```suggestion
Use of a cash register is mandatory in Austria for every company with a total yearly turnover of at least €15,000 and a yearly cash turnover of at least €7,500.
```

> The cash register must comply with the following requirements:

Reviewer: ```suggestion
To ensure that your cash register complies with regulations, take the following steps: 
```

> - Usage of a signature creation unit that ensures that no modification of recorded data can go unnoticed.

Reviewer: ```suggestion
- Use a signature creation unit that ensures that no modification of recorded data can go unnoticed.
```

>   pretixPOS is compatible with the online signature creation unit of A-Trust.

Reviewer: ```suggestion
  pretixPOS is compatible with the  A-Trust online signature creation unit of.
```

> - Registration of the cash register with the tax authorities through FinanzOnline, as well as reporting of every failure of the cash register or signature creation unit.

Reviewer: ```suggestion
- Register the cash register with the tax authorities through FinanzOnline. 
Report every failure of the cash register or signature creation unit.
```

> - Creation of "initial receipts" after registration, "monthly receipts" at the end of every month, "yearly receipts" at the end of every year, and "final receipts" before deprovisioning of the cash register.

Reviewer: ```suggestion
- Create so-called "initial receipts" after registration, "monthly receipts" at the end of every month, "yearly receipts" at the end of every year, and "final receipts" before you decommission the cash register. 
```

>   pretixPOS supports creating these receipts. Note that initial receipts and yearly receipts *must* be scanned using the BMF's [Beleg Check app](https://play.google.com/store/apps/details?id=at.gv.bmf.belegcheck&hl=de).

Reviewer: ```suggestion
  pretixPOS supports creating these receipts. 
  
- Use the BMF's [Beleg Check app](https://play.google.com/store/apps/details?id=at.gv.bmf.belegcheck&hl=de) to scan these receipts. 
```

> - Issuing of receipts with all information required by [§ 11 RKSV](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=20009390) and [§ 11 UStG 1994](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10004873&FassungVom=2024-08-29&Artikel=&Paragraf=11&Anlage=&Uebergangsrecht=).

Reviewer: ```suggestion
- Issue receipts with all information required by [§ 11 RKSV](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=20009390) and [§ 11 UStG 1994](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10004873&FassungVom=2024-08-29&Artikel=&Paragraf=11&Anlage=&Uebergangsrecht=).
```

  pretixPOS allows issuing compliant receipts using a receipt printer.
  Customers are required to take the receipt.

> - Compatibility with a defined export format.

Reviewer: ```suggestion
- Export data in a defined export format.  
```

  pretixPOS is able to export data in the "Datenerfassungsprotokoll" (DEP) format.

>   Note that you are required by law to make a backup of this file at least once every three months.

Reviewer: ```suggestion
  Create a backup of this file at least once every three months. 
  This is required by law. 
```

You can use pretixPOS in compliance with Austrian laws on cash registers ([§ 131b BAO](https://www.ris.bka.gv.at/NormDokument.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10003940&FassungVom=2025-01-27&Artikel=&Paragraf=131b&Anlage=&Uebergangsrecht=), [RKSV](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=20009390)).

Use of a cash register is mandatory in Austria for every company with a total yearly turnover of at least €15,000 and a yearly cash turnover of at least €7,500.
To ensure that your cash register complies with regulations, take the following steps: 

- Use a signature creation unit that ensures that no modification of recorded data can go unnoticed.
  pretixPOS is compatible with the  A-Trust online signature creation unit.

- Register the cash register with the tax authorities through FinanzOnline. 
Report every failure of the cash register or signature creation unit.

- Create so-called "initial receipts" after registration, "monthly receipts" at the end of every month, "yearly receipts" at the end of every year, and "final receipts" before you decommission the cash register. 
  pretixPOS supports creating these receipts. 
  

> - Use the [Beleg Check app](https://play.google.com/store/apps/details?id=at.gv.bmf.belegcheck&hl=de) by Bundesministerium für Finanzen (BMF) to scan these receipts. 

Reviewer: Name der App korrigiert. 
Es gibt auch andere Bezugsquellen als Google, daher würde ich hier lieber auf die offizielle Infoseite des BMF verlinken. 
```suggestion
- Use the app [BMF Belegcheck](https://www.bmf.gv.at/services/apps.html#belegcheck) by Bundesministerium für Finanzen (BMF) to scan these receipts. 
```
